server/cluster_test.go

```diff
@@ -339,59 +339,70 @@ func (s *baseCluster) resetStoreState(c *C, storeID uint64, state metapb.StoreSt
 	c.Assert(err, IsNil)
 }
 
-func (s *baseCluster) testRemoveStore(c *C, clusterID uint64, store *metapb.Store) {
+func (s *baseCluster) testStateAndLimit(c *C, clusterID uint64, store *metapb.Store, beforeState metapb.StoreState, run func(*RaftCluster) error, expectStates ...metapb.StoreState) {
+	// prepare
+	storeID := store.GetId()
 	cluster := s.getRaftCluster(c)
+	cluster.coordinator.opController.SetAllStoresLimit(1.0)
+	s.resetStoreState(c, store.GetId(), beforeState)
+	_, beforeLimit := cluster.coordinator.opController.GetAllStoresLimit()[storeID]
```

Comment:
Since it is a bool variable, the name is a little bit wired.

Reply:
how about `isOKBefore`?